src/App.tsx

```diff
@@ -94,18 +82,16 @@ const checkoutTheme = createTheme({
     MuiLoadingButton: {
       styleOverrides: {
         loadingIndicator: {
-          color: "white !important",
+          color: "white",
+          size: "24px",
```

Comment:
Size appear to not be a valid properties for this element, you could try setting width and height